lib/myfinance/resources/receivable_account.rb

```diff
@@ -9,26 +9,20 @@ module Resources
     class ReceivableAccount < FinancialAccount
 
       def receive(id, entity_id, params = {})
-        http.put(endpoints(id, entity_id, :receive), body: { receivable_account: params }) do |response|
-          respond_with_object(response, 'receivable_account')
-        end
+        request_and_build_response(:put, endpoint_for(id, entity_id, :receive), params)
       end
 
       def undo_receivement(id, entity_id)
-        http.put(endpoints(id, entity_id, :undo_receivement)) do |response|
-          respond_with_object(response, 'receivable_account')
-        end
+        request_and_build_response(:put, endpoint_for(id, entity_id, :undo_receivement))
       end
 
       private
 
-      def endpoints(id, entity_id, key)
-        {
-          create: '/entities/:entity_id/receivable_accounts',
+      def endpoints
+        default_endpoints.merge({
           receive: '/entities/:entity_id/receivable_accounts/:id/receive',
           undo_receivement: '/entities/:entity_id/receivable_accounts/:id/undo_receivement',
-          destroy: '/entities/:entity_id/receivable_accounts/:id'
-        }[key].gsub(':entity_id', entity_id.to_s).gsub(':id', id.to_s)
+        })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.